packages/mwp-rasp-plugin/src/sigsci/index.js

```diff
@@ -533,24 +536,19 @@ Sigsci.prototype.onPostResponse = function(err /* , rpcResponse */) {
 // HAPI.JS Support
 // ------------------------------------
 
-Sigsci.prototype.hapi = function() {
```

Comment:
removed this function because it boils down directly into `Sigsci.middlewareHapi`